kcidb/test_orm.py

```diff
@@ -498,3 +500,637 @@ def test_repr(self):
+                        "status": "PASS",
+                        "id": "kernelci:kernelci.org:619c656de1fb4af479f2efae",
+                        "build_id": "kernelci:kernelci.org:619c64b171"
+                                    "2847eccbf2efac",
+                        "origin": "kernelci"
+                    },
+                    {
+                        "build_id": "kernelci:kernelci.org:619c65f9709"
+                                    "de72e90f2efd0",
+                        "id": "kernelci:kernelci.org:b9d8be63bc2abca6316"
+                              "5de5fd74f0f6d2f0b0d1c",
+                        "origin": "kernelci"
+                    },
+                    {
+                        "build_id": "redhat:redhat.org:619c65f970"
+                                    "9de72e90f2efd0",
+                        "id": "redhat:redhat.org:b9d8be63bc2abca63165"
+                              "de5fd74f0f6d2f0b0d1c",
+                        "origin": "redhat",
+                        "waived": True,
+                        "status": "DONE"
+                    },
+                    {
+                        "build_id": "kernelci:kernelci.org:619c65d3c1"
+                                    "b0a764f3f2efa0",
+                        "id": "kernelci:kernelci.org:b9d8be63bc2abca63165"
+                              "de5fd74f0f6d2f0b0e2b",
+                        "origin": "kernelci",
+                        "status": "PASS",
+                        "waived": False,
+                        "start_time": "2021-11-23T03:52:13.666000+00:00",
+                        "path": "baseline.dmesg.crit"
+                    }
+                ],
+                "checkouts": [
+                    {
+                        "id": "_:kernelci:5acb9c2a7bc836e"
+                              "9e5172bbcd2311499c5b4e5f1",
+                        "origin": "kernelci",
+                        "git_commit_hash": "5acb9c2a7bc836e9e5172bb"
+                                           "cd2311499c5b4e5f1",
+                        "git_commit_name": "v5.15-4077-g5acb9c2a7bc8",
+                        "patchset_hash": ""
+                    },
+                    {
+                        "id": "_:kernelci:82bcf49e5e6ad570ff61ffcd"
+                              "210cf85c5ec8d896",
+                        "origin": "kernelci",
+                        "patchset_hash": ""
+                    },
+                    {
+                        "id": "_:redhat:5acb9c2a7bc836e9619c65f9"
+                              "709de72e90f2efd0",
+                        "origin": "redhat",
+                        "patchset_hash": ""
+                    }
+                ]
+            }
+        )
+
+    def setUp(self):
+        """Database setup and oo client creation"""
+        self.source = kcidb.db.Client('sqlite::memory:')
+        self.source.init()
+
+        self.initialize_and_load_data()
```

Comment:
This doesn't initialize the source, only loads data, and is probably due for a rename.